### ISSUE TITLE
Fastload

### ISSUE THREAD

**Issue by Madis0** [Fabulously-Optimized/fabulously-optimized#365]
### CurseForge link

https://www.curseforge.com/minecraft/mc-mods/fastload

### CurseForge Mod Distribution

Allowed

### Modrinth link

https://modrinth.com/mod/fastload

### Source/other link

https://github.com/BumbleSoftware/Fastload

### What it does

Reduces world loading time, similar to #160 

### Why should it be in the modpack

Claims to be better than #160

### Why shouldn't it be in the modpack

Untested

### Categories

Performance optimization

**Comment by RaptaG** [comment]
It is [on Modrinth](https://modrinth.com/mod/fastload) actually.

**Comment by Madis0** [comment]
Thanks, updated

**Comment by Kichura** [comment]
Can confirm this actually is way better than kennytv's force-close-loading-screen + ksyxis as it seems to be reducing loading times by a ton on my end.

**Comment by bensuperpc** [comment]
Fastload works fine for me, world loading times are much faster

Tested on Linux (Manjaro) with Fabulously Optimized 4.4.3

I haven't tested Ksyxis

**Comment by RaptaG** [comment]
> I haven't tested Ksyxis

And you don't have to. It's not considered for FO anymore. You can do it for comparison though, if you want to.

**Comment by code004Accepted** [comment]
Video testing Fastload: https://clipchamp.com/watch/a0L5xgk86GZ

TL;DR:

Without Fastload, world generation time: 81 secs, load time: 18 secs

With Fastload, world generation time: 59 secs, load time: 11 secs

**Comment by RozeFound** [comment]
It's 1.19.3 already

**Comment by Madis0** [comment]
Can still not reproduce the improvements on my device.

**Comment by RozeFound** [comment]
> Can still not reproduce the improvements on my device.

Maybe your machine is too powerful to notice impact?

**Comment by Madis0** [comment]
No, the world loading still takes 10-20 seconds for me, so reducing that would be great.

**Comment by RaptaG** [comment]
Make the config like this:
![2023-04-21_18 14 22](https://user-images.githubusercontent.com/77157639/233672710-18db5a1d-559a-46b0-8363-5a765f03d42d.png)
Blazing fast

**Comment by RaptaG** [comment]
@Madis0

**Comment by Madis0** [comment]
> I (StockiesLad) have quit modding again. This will explain the absence of updates. Why again? It's just not fun anymore and I am always worried about once again being embedded in annoying discord drama that somehow manages to suck every bit of enjoyment out of modding. There hasn't been anyone willing to step up and take over development and that's partly my fault. This notice is months late. If you enjoy fastload and intend to maintain it, just contact me on discord for a discussion. Please keep in mind that I won't respond quickly because I don't really use discord much at all anymore. Thank you for taking your time to read this!
